components/net/tests/fetch.rs

```diff
@@ -397,8 +399,8 @@ fn test_cors_preflight_cache_fetch() {
     assert_eq!(1, counter.load(Ordering::SeqCst));
 
     // The entry exists in the CORS-preflight cache
-    assert_eq!(true, cache.match_method(&wrapped_request0, Method::GET));
-    assert_eq!(true, cache.match_method(&wrapped_request1, Method::GET));
+    assert_eq!(true, cache.match_method(&wrapped_request2, Method::GET));
+    assert_eq!(true, cache.match_method(&wrapped_request3, Method::GET));
```

Comment:
I think you're breaking this test now. Aren't `wrapped_request2` and `wrapped_request3` the same? (Whereas `wrapped_request0` and `wrapped_request1` were potentially different before?)

Reply:
I don't think so. The request doesn't change in that call. In the calls to `fetch_with_cors_cache` on lines `392` and `393` the `cache` is passed in as a mutable reference and I think the assert is checking the cache.

In the [match method](https://github.com/servo/servo/blob/2115b6a6db33de8c86b6c4899926c7f0923825a0/components/net/fetch/cors_cache.rs#L74) call, it makes a comparison between the cors cache and the request's `origin`, `current_url()` and checks the `credentials_mode`. Hopefully I didn't miss anything but the `fetch` calls do not make modifications to these values.

I did rename the requests to be `wrapped_request0_copy`, for example, which might be easier to understand?

Reply:
Ah I think you're right. The second assert looks redundant to me, but it was this way before so let's keep it as-is for now.